core-web/libs/dotcms-models/src/lib/dot-experiments.model.ts

```diff
@@ -112,16 +112,23 @@ export type GoalsLevels = 'primary';
 export interface Goal {
     name: string;
     type: GOAL_TYPES;
-    conditions?: Array<GoalCondition>;
+    conditions: Array<UrlParameterGoalCondition | ReachPageGoalCondition>;
 }
 
 export type Goals = Record<GoalsLevels, Goal>;
 
-export interface GoalCondition {
+interface ReachPageGoalCondition {
     parameter: GOAL_PARAMETERS | string;
-    operator: GOAL_OPERATORS;
+    operator: GOAL_OPERATORS | null;
```

Comment:
it will be `null`? or can we just make it `operator?' as a optional property?

Reply:
No, I need to remove the null. tnks

Reply:
Removed.